nss-cohort-40/c40-keahua-arboretum-da_beach_combers
# Report

# Description
Cultivation functionality added.
Add len function to release animals to give feedback of number of animals
Fixes # (Added name to Environment to dynamically display name of Environment)
        # (Changed plants to plant for display of report)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
Create new Environments and Add animals to test if it shows you how many animals are in each Biome.
Create new Environments and Add plants to each environment.
Report at the end to see where plants were added properly.
# Checklist:
